lib/nexpose/scan.rb

```diff
@@ -763,6 +788,42 @@ def self._parse_status(code)
         :stopped
       when 'A'
         :aborted
+      when 'P'
+        :paused
+      when 'I'
```

Comment:
Need to remove these statuses from the `CompletedScan` class since it's only useful to `ActiveScan`.